app/Mage.php

```diff
@@ -964,14 +964,8 @@ public static function getIsDeveloperMode()
      */
     public static function printException(Throwable $e, $extra = '')
```

Comment:
This is also called in cron context.

Reply:
interesting, but it's up to the external module to handle that situation so it shouldn't be a problem here

Reply:
Should it? Yea maybe. Would be the first time tho. None other event to my knowledge just gets executed and the listener has to check if its from cli.

Reply:
Good point. What do we have to do in this situation? Catch in ignition could be useful if we want to track in flare (not implemented yet) as well, but the output is not CLI-friendly.
I think it's a situation to handle in the external module.

Reply:
well in this case, since the error could be sent to Flare or another log collecting service... probably it's ok that the event triggers also in the cron environment

Reply:
Regarding this PR, the event must be dispatched; then, the logic within the observer will must to handle it